ui/shared/RecipientSelector.qml

```diff
@@ -79,7 +93,7 @@ Item {
             Layout.fillWidth: true
             anchors.left: undefined
             anchors.right: undefined
-            //% "No recipient selected"
+	    //% "No recipient selected"
```

Comment:
Maybe it was the translation script that did those? 🤔 
```suggestion
	      //% "No recipient selected"
```

Reply:
Fixed.